service/src/main/kotlin/no/nav/su/se/bakover/service/kontrollsamtale/KontrollsamtaleService.kt

```diff
@@ -0,0 +1,111 @@
+package no.nav.su.se.bakover.service.kontrollsamtale
+
+import arrow.core.Either
+import arrow.core.getOrElse
+import arrow.core.left
+import arrow.core.right
+import no.nav.su.se.bakover.client.person.MicrosoftGraphApiOppslag
+import no.nav.su.se.bakover.common.Tidspunkt
+import no.nav.su.se.bakover.common.log
+import no.nav.su.se.bakover.common.persistence.SessionFactory
+import no.nav.su.se.bakover.domain.NavIdentBruker
+import no.nav.su.se.bakover.domain.Person
+import no.nav.su.se.bakover.domain.brev.LagBrevRequest
+import no.nav.su.se.bakover.domain.dokument.Dokument
+import no.nav.su.se.bakover.domain.oppgave.OppgaveConfig
+import no.nav.su.se.bakover.service.brev.BrevService
+import no.nav.su.se.bakover.service.brev.KunneIkkeLageBrev
+import no.nav.su.se.bakover.service.oppgave.OppgaveService
+import no.nav.su.se.bakover.service.person.PersonService
+import no.nav.su.se.bakover.service.sak.SakService
+import java.time.Clock
+import java.time.LocalDate
+import java.util.UUID
+
+interface KontrollsamtaleService {
+    fun kallInn(sakId: UUID, saksbehandler: NavIdentBruker): Either<KunneIkkeKalleInnTilKontrollsamtale, Unit>
+}
+
+class KontrollsamtaleServiceImpl(
+    private val sakService: SakService,
+    private val personService: PersonService,
+    private val brevService: BrevService,
+    private val oppgaveService: OppgaveService,
+    private val microsoftGraphApiOppslag: MicrosoftGraphApiOppslag,
+    private val sessionFactory: SessionFactory,
+    private val clock: Clock,
+) : KontrollsamtaleService {
+
+    override fun kallInn(
```

Comment:
Helt generelt her; kunne/burde vi innført et konsept for kontrollsamtale i samme slengen? Hele domenemodellen er vel ikke kjent enda, men det er vel fair å anta at alle kontrollsamtaler kommer til å starte med en innkalling? I så fall kunne vi koblet dokument og oppgave mot den aktuelle "kontrollsamtalen". Videre kunne vi utvidet dette objektet med funksjonalitet for å håndtere "det som skjer etter" kontrollsamtalen?

Reply:
Kommentar under.